app/scripts/lib/AccountIdentitiesPetnamesBridge.test.ts

```diff
@@ -97,59 +90,22 @@ function createNameStateWithPetname(
   };
 }
 
-const EMPTY_ACCOUNTS_CONTROLLER_STATE: AccountsControllerState = {
-  internalAccounts: { accounts: {}, selectedAccount: '' },
-};
-
-function setupAccountsController(
-  messenger: AccountsControllerMessenger,
-  state = EMPTY_ACCOUNTS_CONTROLLER_STATE,
-) {
-  return new AccountsController({ state, messenger });
-}
-
-function buildKeyringControllerMessenger(
-  messenger,
-): KeyringControllerMessenger {
-  return messenger.getRestricted({
-    name: 'KeyringController',
-    allowedEvents: ['KeyringController:stateChange'],
-    allowedActions: ['KeyringController:getKeyringsByType'],
-  });
-}
-
-function buildAccountsControllerMessenger(
-  messenger,
-): AccountsControllerMessenger {
-  return messenger.getRestricted({
-    name: 'AccountsController',
-    allowedEvents: [
-      'SnapController:stateChange',
-      'KeyringController:accountRemoved',
-      'KeyringController:stateChange',
-    ],
-    allowedActions: [
-      'KeyringController:getAccounts',
-      'KeyringController:getKeyringForAccount',
-      'KeyringController:getKeyringsByType',
-    ],
-  });
-}
-
-function buildPetNamesBridgeMessenger(
-  messenger,
-): PetnamesBridgeMessenger<
-  AccountIdentitiesPetnamesBridgeEvents,
-  AccountIdentitiesPetnamesBridgeActions
+function createMessengerMock(): jest.Mocked<
+  PetnamesBridgeMessenger<
```

Comment:
If you're using this type a lot it can help to define it somewhere to avoid repetition.
```
type BridgeMessenger = PetnamesBridgeMessenger<
    AccountIdentitiesPetnamesBridgeEvents,
    AccountIdentitiesPetnamesBridgeActions
  >;
```